src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -377,7 +378,7 @@ function ReportActionCompose({
                 {shouldShowReportRecipientLocalTime && hasReportRecipient && <ParticipantLocalTime participant={reportRecipient} />}
             </OfflineWithFeedback>
             <View style={isComposerFullSize ? styles.flex1 : {}}>
-                <PortalHost name="suggestions" />
+                <PortalHost name="suggestions_0" />
```

Comment:
This ("suggestions_0" string) would get lost during a refactor, it's not clear why we have this specific name here. I propose we create a new key in CONST, use it here and then in `src/components/AutoCompleteSuggestions/index.native.tsx` we could do something along the lines of `<Portal hostName={ activeID ? "suggestions_${activeID}" : CONST.DEFAULT_COMPOSER_PORTAL_HOST_NAME}>`